app/controllers/concerns/verify_profile_concern.rb

```diff
@@ -9,7 +9,6 @@ def account_or_verify_profile_url
   end
 
   def user_needs_backup_code_reminder?
-    return false unless IdentityConfig.store.backup_code_reminder_redirect
```

Comment:
if this is no longer used, can we remove it from `lib/identity_config.rb` and `config/application.yml.default`?

Reply:
Yes, thank you! I have made those changes.

First commit was to see if the tests would pass. Now I believe the PR is complete.